pipelines/h3c/rtp/v1/task_scripts/do_ADD_RTP_PROCESS_RECORD.sh

```diff
@@ -14,10 +14,18 @@ fn="${1}"
 # to pipeline_list
 shift
 pipeline_list="$@"
-# string-ify the arguments joining with ''
-# then remove "[", "]", "SETUP,", ",TEARDOWN", and ",ADD_RTP_PROCESS_RECORD"
-action_string=$(join_by ''  ${pipeline_list[@]} | sed -E "s/\[|\]|(SETUP,)|(,TEARDOWN)|(,ADD_RTP_PROCESS_RECORD)//g")
 
+# Diff files did not have the ANT metrics phase run on them
```

Comment:
I'm a little concerned there isn't a more automated way to do this, but I suppose as long as we remember to update this task script to reflect what's been run on the sum vs. diff files, we should be fine.

Reply:
Yeah I would prefer if there were an automated way, or something in the toml to specify diff_actions or diff_skip actions and the make_rtp_makeflow could parse that. But that sounds like it could be some work.